content/sensu-go/5.19/reference/agent.md

```diff
@@ -740,6 +740,8 @@ Flags:
       --annotations stringToString            entity annotations map (default [])
       --api-host string                       address to bind the Sensu client HTTP API to (default "127.0.0.1")
       --api-port int                          port the Sensu client HTTP API listens on (default 3031)
+      --assets-burst-limit int                asset fetch burst limit (default 100) -- Sensu Go 5.19.3
+      --assets-rate-limit float               maximum number of assets fetched per second -- Sensu Go 5.19.3
```

Comment:
I think it's fair to note that these flags are introduced in 5.19.3, but maybe just not in the help usage. Would it make sense to make a note about that somewhere else in the reference?